networkx/algorithms/planarity.py

```diff
@@ -761,6 +783,14 @@ class PlanarEmbedding(nx.DiGraph):
     For a half-edge (u, v) that is orientated such that u is below v then the
     face that belongs to (u, v) is to the right of this half-edge.
 
+    Note
+    ----
+
+    It is suggested to use `check_planarity()` function first, since it is considered as the
+    primary interface to the features of this class as it returns a PlanarEmbedding
+    if the input graph is planar. Returned embedding can then be used to apply other methods
+    of the class.
+
```

Comment:
This is a very nice, detailed note but IMO it's more important to have the relevant info higher up in the docstring. Lots of readers don't make it all the way down to the `Notes` section on a first read!

I'd recommend something like adding:

```
`check_planarity` is the preferred way to check if a graph is planar.
```

at the top of the docstring just before the **Neighbor ordering** section. It might even be worth putting this info in a `.. note::` directive to really draw extra attention to it.

I would also advocate here that we replace this `Note` section with a `See Also` section that links back to `check_planarity`, e.g. something like:

```suggestion
    See Also
    --------
    check_planarity : 
        Preferred way to check if an existing graph is planar. Also a more
        convenient way to create a `PlanarEmbedding` from an existing graph.

```